src/app/frontend/poddetail/containerinfo.html

```diff
@@ -56,5 +56,11 @@
         {{::$ctrl.i18n.MSG_CONTAINER_DETAILS_NO_ARGS}}
       </div>
     </kd-info-card-entry>
+    <kd-info-card-entry>
```

Comment:
AFAIK containerinfo.html is also used from replication controllers and others. You should do checking whether podName is specified.